tests/cephfs/cephfs_nfs/test_cephfs_nfs_ha_node_replacement.py

```diff
@@ -0,0 +1,367 @@
+            raise CommandFailed("Keepalived services not started BZ : 2240258")
+        log.info(
+            f"The active node changed after Power off."
+            f"\nPrevious active node: {active_nfs.node.hostname}"
+            f"\nCurrent active node: {active_nfs_after.node.hostname}"
+        )
+        active_nfs_ipmi = IPMIPowerControl(
+            host=f"{backend_server[0].node.hostname}.ipmi.ceph.redhat.com",
+            username="inktank",
+            password="ApGNXcA7",
+        )
+        active_nfs_ipmi.power_up()
+        fs_util_v1.wait_for_host_online(client1, backend_server[0])
+        ps_ls = client1.exec_command(
+            sudo=True, cmd=f"ceph orch ps {backend_server[0].node.hostname} -f json"
+        )
+        if "nfs" in ps_ls:
+            raise CommandFailed(
+                f"Still Holding the nfs services on the removed server {backend_server[0].node.hostname}"
+            )
+
+        backend_server = get_active_nfs_server(client1, nfs_name, ceph_cluster)
+        for server in backend_server:
+            log.info(f"Active backend servers after reboots {server.node.hostname}")
+        log.info("Test completed successfully")
+        return 0
+    except Exception as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
+    finally:
+        log.info("Cleaning Up")
+        client1.exec_command(
+            sudo=True, cmd=f"rm -rf {nfs_mounting_dir}*", check_ec=False
+        )
+        log.info("Unmount NFS export")
+        client1.exec_command(
+            sudo=True, cmd=f"umount -l {nfs_mounting_dir}", check_ec=False
+        )
+        log.info("Removing the Export")
+        client1.exec_command(
+            sudo=True,
+            cmd=f"ceph nfs export delete {nfs_name} {nfs_export_name}",
+            check_ec=False,
+        )
+        log.info("Removing NFS Cluster")
+        client1.exec_command(
+            sudo=True,
+            cmd=f"ceph nfs cluster rm {nfs_name}",
+            check_ec=False,
+        )
+        log.info("Powering on the server")
+        active_nfs_ipmi = IPMIPowerControl(
+            host=f"{backend_server[1].node.hostname}.ipmi.ceph.redhat.com",
+            username="inktank",
+            password="ApGNXcA7",
+        )
+        active_nfs_ipmi.power_up()
+        active_nfs_ipmi = IPMIPowerControl(
+            host=f"{backend_server[0].node.hostname}.ipmi.ceph.redhat.com",
+            username="inktank",
+            password="ApGNXcA7",
+        )
+        active_nfs_ipmi.power_up()
+        fs_util_v1.wait_for_host_online(client1, backend_server[1])
+        fs_util_v1.wait_for_host_online(client1, backend_server[0])
+
+
+def get_active_nfs(nfs_servers, virtual_ip):
+    for nfs in nfs_servers:
+        out, rc = nfs.exec_command(sudo=True, cmd="ip a")
+        if virtual_ip in out:
+            return nfs
+
+
+def get_active_nfs_server(client, nfs_cluster, ceph_cluster):
```

Comment:
These two modules are added to fsutils_v1 testlib through https://github.com/red-hat-storage/cephci/pull/2966/files#top
Do we still need to have a copy here?

Reply:
Shall i refactor this once the PR gets merged.